packages/material-design-icons/src/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+	STATIC_PATH: __dirname,
```

Comment:
is this used by anything?

Reply:
I pulled that from the official material-design-icons package.  My thought was that it might make it easier for folks to move from that package to this alternative (either in full or just to get SVGs missing from the official package), if they were relying on that for path resolution.